examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,319 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water through a
+sharp-edged orifice. This is a typical configuration in fuel injectors, and
+brings a challenge to the physics and numerics of cavitation models because of
+the high pressure differentials involved and the high ratio of liquid to vapor
+density.
+
+This example uses the multiphase modeling capability of Ansys Fluent, and will
+be able to predict the strong cavitation near the orifice after flow separation
+at a sharp edge.
+
+**Workflow tasks**
+
+This example demonstrates how to do the following:
+
+- Set boundary conditions for internal flow
+- Use the mixture model with cavitation effects
+- Calculate a solution using the pressure-based coupled solver
+
+**Problem description**
+
+The problem considers the cavitation caused by the flow separation after a
+sharp-edged orifice. The flow is pressure driven, with an inlet pressure of 500
+kPa and an outlet pressure of 95 kPa. The orifice diameter is 4 mm, and the
+geometrical parameters of the orifice are D/d = 2.88 and L/d = 4, where D, d,
+and L are the inlet diameter, orifice diameter, and orifice length respectively.
+"""
+
+###############################################################################
+# Example Setup
+# -------------
+# Before you can begin, you must set up the example and initialize this
+# workflow.
+#
+# Launch Fluent
+# ~~~~~~~~~~~~~
+# Perform required imports, which includes downloading and importing
+# the geometry file.
+
+# sphinx_gallery_thumbnail_path = '_static/cavitation_model_thumb.png'
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+cav_file = examples.download_file("cav.msh", "pyfluent/cavitation")
+
+###############################################################################
+# Launch Fluent session in 2d solution mode with double precision running
+# on four processors.
+
+solver = pyfluent.launch_fluent(
+    precision="double",
+    processor_count=4,
+    mode="solver",
+    version="2d",
+)
+
+###############################################################################
+# Read the Mesh and Specify Models
```

Comment:
The intention of my previous suggestion was to turn this into a heading, like so:
```suggestion
# Read the Mesh and Specify Models
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
If you prefer this as text rather than a heading, then you should use sentence case and not title case.